api/views/maps.py

```diff
@@ -0,0 +1,56 @@
+from api import app, db
+from api.models import Map, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request, jsonify
+import json
+#from api.utils import InvalidUsage
+# from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+# from flask_sqlalchemy import SQLAlchemy
+
+MAPS_URL = '/maps'
+MAPS_ID_URL = '/maps/<int:map_id>'
+
+mod = Blueprint('maps', __name__)
+
+#none of these have been comprehsnsively tested yet
+
+# Get all maps.
```

Comment:
and these, the code below is pretty self documenting, so these comments are redundant